class.photon.php

```diff
@@ -720,7 +718,11 @@ public function filter_srcset_array( $sources, $size_array, $image_src, $image_m
 					);
 			} // foreach ( $multipliers as $multiplier )
 			if ( is_array( $newsources ) ) {
-				$sources = array_replace( $sources, $newsources );
+				if ( function_exists( 'array_replace' ) ) { // PHP 5.3+, preferred
+					$sources = array_replace( $sources, $newsources );
```

Comment:
In retrospect, I'm reversing myself.  Slightly more complicated logic results in better performance.